### DISCUSSION TITLE
[@types/yt-search] PlaylistItem interface is missing a duration field

### DISCUSSION THREAD

**Discussion by gulebi** [DefinitelyTyped/DefinitelyTyped#66403] [Issues with a @types package]
yt-search: 2.10.4
@types/yt-search: 2.3.2

There is no field in the interface, although the library provides a duration.

PlaylistItem interface:
```
interface PlaylistItem {
  title: string;
  videoId: string;
  listId: string;
  thumbnail: string;
  author: Author;
}
```
Proposed fix:
```
interface PlaylistItem {
  title: string;
  videoId: string;
  listId: string;
  thumbnail: string;
  author: Author;
  duration: Duration;
}
```

[a comment by an automated account is not shown]

**Comment by cherryblossom000** [comment]
The `duration` field was [added in yt-search v2.5.0](https://github.com/talmobi/yt-search/issues/35). The current types of yt-search are for v2.3, and the latest version of yt-search is v2.10.4.

I just opened a PR to fix this as well as well as fixing some other issues: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/66419

~~I think it’s better to not only add the duration field but to also ensure the other types are updated for v2.10. I am very busy at the moment so I probably won’t be able to update the typings for the latest version of yt-search soon, but feel free to open a PR if you’re willing to do this!~~

~~Here’s [the diff](https://github.com/talmobi/yt-search/compare/fc27597fd83ce259b01c37d1ec0329493d5b69e1...master#diff-bfe9874d239014961b1ae4e89875a6155667db834a410aaaa2ebe3cf89820556) since I first added the types.~~
